### PR TITLE
Docs: Top menu structure change

### PR DESCRIPTION
**Pull request by elitan**
Current entry point to our documentation and menus:

image

New:

image

What changed:
- Removed Get Started top menu and content
- Renamed Platform to Documentation (the logo links to our landing page)

Reasoning:

- Only one introduction page
- See all features from the landing page (Database, GraphQL, Authentication, etc)
- See quickstart in various frameworks from the landing page

Currently, we have the following under "Get Started":

- General quickstart guide
- CLI guide
- Upgrade from Nhost v1 to Nhost v2.

I propose that we move the general quickstart to a blog post for now. The current quickstart guide does not follow the same framework format that we have under the new "Quickstarts" section (React, Next.js, etc).

The CLI guide should be removed in favor of https://docs.nhost.io/platform/overview/get-started-with-nhost-cli. That "Get started with Nhost CLI" does not fit under "Overview" IMO but that's a different topic.

Move "Upgrade from v1 to v2" to its own blog post with a better SEO title. Most projects have already been migrated.

### CODE REVIEW

**Review by szilarddoro — CHANGES_REQUESTED** [review]

File: docs/vercel.json
Comment (elitan) [reply docs/vercel.json:3]: Do you mean the old URLs from `/get-started/*`? Or some other URLs?

The URLs that were previously under Platform are still under `/platform/*`.

Comment (guicurcio) [reply docs/vercel.json:3]: We should try to preserve old links (or add redirects) as much as possible, especially since we have many websites linking to these tutorials and we don't want to show them a 404 or lose the link equity: e.g. this [stackoverflow question](https://stackoverflow.com/questions/71526371/nhost-typeerror-intermediate-value-is-not-a-function) links to the following page: https://docs.nhost.io/get-started/quick-start/javascript-client which would show a 404 with this PR.

Comment (elitan) [reply docs/vercel.json:3]: Added redirects for the CLI URLs. I'll add redirects for the quickstart guide and migrations tutorial once this PR is merged and the two blog posts are up.

**Review by nunopato — APPROVED** [review]

**Review by plmercereau — APPROVED** [review]

**Comment by plmercereau** [comment]
the redwoodjs disappeared in the preview
https://docs-gbyszevox-nhost.vercel.app/

**Comment by elitan** [comment]
All should be good now. I need another approval to merge. If you see this, please approve and merge.

**Review by plmercereau — APPROVED** [review]

**Pull request merged**
